product_docs/docs/migration_portal/3.5.0/04_mp_migrating_database/03_mp_schema_migration.mdx

```diff
@@ -166,30 +143,22 @@ Migrate schemas to an on-premises EDB Postgres Advanced Server on Linux.
 
 1.  Select **New On-premises EDB Postgres Advanced Server**.
 
-![Migrating schemas home page](../images/mp_schema_migration_home_new.png)
 
 2.  Select one or more schemas to migrate on EDB Postgres Advanced Server.
 
-![Selecting schemas for migration](../images/mp_schema_mig_new_epas_schemas_selection.png)
-
 3.  For the operating system, select the Linux.
 
-![Selecting Linux operating system](../images/mp_schema_mig_new_epas_linux.png)
 
 4.  You can select one of the following options to install the EDB Postgres Advanced Server:
 
     -   Repository
     -   More options
 
-    ![Selecting Linux repository](../images/mp_schema_mig_new_epas_linux_repo.png)
-
 5.  For information on the installation procedure, select `EDB Postgres Advanced Server Installation Guide` for Linux.
```

Comment:
Is this a link on the UI? If so, use bold not code formatting.

Reply:
Fixed.